squad/core/notification.py

```diff
@@ -113,8 +113,11 @@ def __send_notification__(project, notification):
         context=context,
     )
     sender = "%s <%s>" % (settings.SITE_NAME, settings.EMAIL_FROM)
-    for r in recipients:
-        message = EmailMultiAlternatives(subject, text_message, sender, [r.email])
-        if r.html:
-            message.attach_alternative(html_message, "text/html")
-        message.send()
+
+    emails = [r.email for r in recipients]
+    use_html = all([r.html for r in recipients])
```

Comment:
Does this mean that nobody will receive html email if any one recipient does not want html email?